packages/forms/src/elements/select/select.tsx

```diff
@@ -109,7 +108,6 @@ export const Select: React.FC<SelectProps & FieldRenderProps<string>> = ({
 							{isOpen && (
 								<PopupBox
 									{...{
-										searchable,
```

Comment:
removing `searchable` breaks the ability to search in the `FFSelect` input when `readOnly={false}`.
We can avoid creating a variable, but we need to pass the `searchable` prop to `PopupBox`, otherwise will always contain the default value of `false`. We can do something like:
```
<PopupBox
	searchable = {!readOnly}
	{...{
		getItemProps,
		inputValue,
		options,
		highlightedIndex,
		selectedItem,
		handleNotFoundButtonClick,
		notFoundMessage,
	}}
/>
```

Reply:
That's how it was before when you only saw one result. That came from passing searchable = true. It only works if it receives false or undefined. So, yes, this is a bug with FFSelect but it was there anyway and I think we can log it separately outside of MVP. Do you agree?